src/components/Support/form/NewRequestForm.tsx

```diff
@@ -52,6 +57,39 @@ function NewRequestForm() {
     await submitForm(formValues, resetFormValues);
   };
 
+  async function fetchHealthStatus() {
+    setIsLoading(true);
+    try {
+      const healthStatus = await checkHealthStatus();
+      setIsHealthy(healthStatus);
+      if (!healthStatus) {
+        toast.error(
+          'Our support system is currently experiencing issues. Please report this to our team.',
+        );
+      }
+    } catch (error) {
+      toast.error('Failed to fetch health status. Please try again later.');
```

Comment:
The user shouldn't be concerned by a "health status". As a user, imagine that you're reporting an issue and you'd face "Failed to fetch health status. Please try again later.". So, if the health checkup fails, we let know something happened and we want to do something about it.

```suggestion
      if (!healthStatus) {
         throw new Error('Health status failure')
      }
    } catch (error) {
       console.error(error.message);
        toast.error(
          'Our support system is currently experiencing issues. Please report this to our team.',
        );
```

Reply:
Valid! Should have just duplicated this.